lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -166,7 +166,7 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     }
 
     // Budgets
-    const budgetAudit = category.auditRefs.find(audit => audit.id === 'performance-budget');
```

Comment:
can we leave this backwards compatible so it looks for either?

Reply:
this also reveals the fragility of searching for this based on the audit id I guess :/ I wonder what the harm would be of duplicating this audit result under the `performance-budget` id

just cruft and a larger LHR I suppose?